package.nls.json

```
@@ -25,6 +25,7 @@
     "AWS.configuration.description.samcli.debug.attach.timeout": "Maximum time (in milliseconds) to wait for SAM output while starting a Local Lambda session",
     "AWS.configuration.description.samcli.location": "Location of SAM CLI. SAM CLI is used to create, build, package, and deploy Serverless Applications. [Learn More](https://aws.amazon.com/serverless/sam/)",
     "AWS.configuration.description.telemetry": "Enable AWS Toolkit to send usage data to AWS.",
+    "AWS.configuration.suppressCodeLenses": "Suppress SAM hints in source files",
```

positive logic, common terminology
```suggestion
    "AWS.configuration.enableCodeLenses": "Enable SAM hints in source files",
```